src/cozy/wrappers/confirm-your-identity-dialog/confirm-your-identity-dialog.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ReactWrapper, { reactWrapperProps } from '../react-wrapper';
+
+import Button from 'cozy-ui/transpiled/react/Button'
+import { FixedDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+import './styles.css'
+
+const ConfirmYourIdentityModal = ({ 
+  ownerName,
+  fingerprintPhrase,
+  onClose
+}) => {
+  const { t } = useI18n()
+  
+  let dialogTitle = t(`ConfirmYourIdentityModal.title`)
+
+  const instruction1 =  t(`ConfirmYourIdentityModal.instruction`, {
+    ownerName: ownerName
+  })
+
+  const instruction2 = t(`ConfirmYourIdentityModal.instruction2`, {
+    ownerName: ownerName
+  })
+
+  let dialogContent = (
+    <>
+      <div>
+        <p>{instruction1}</p>

```

Comment:
We should not use `<p>` `<span>` or else. `<Typography>` is the way to go

Reply:
Fixed